tests/suite/manifests/nginxgateway/nginx-gateway.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: gateway.nginx.org/v1alpha1
+kind: NginxGateway
+metadata:
+  annotations:
```

Comment:
This is an example nginx-gateway yaml file that works for the test, but there are many places like the labels and the release-name which can be broken so easily, but I don't see another option. Anyone have ideas?

Reply:
You don't need to supply any of the labels or annotations when installing this.

Reply:
Removed, do you think leaving the name statically set is okay? It currently is just the release name + config.

Reply:
I think it's fine.